code/modules/projectiles/guns/energy/special.dm

```diff
@@ -200,6 +201,7 @@
 	name = "bluespace wormhole projector"
 	desc = "A projector that emits high density quantum-coupled bluespace beams. Requires a bluespace anomaly core to function."
 	ammo_type = list(/obj/item/ammo_casing/energy/wormhole, /obj/item/ammo_casing/energy/wormhole/orange)
+	w_class = WEIGHT_CLASS_BULKY
```

Comment:
I don't think portal gun should be bulky, its utility item, not a gun

Reply:
it's in the guns area, it's a gun. It doesn't do damage, but it's large and makes sense to be bulky.

Reply:
First of all, it requires bluespace core. Second of all, it is utility item. Having big sprite does not make it bulky. Its rarely used, now it will be used even less.

Reply:
I'll leave this unresolved as I'm not sure which way I want to go with it. I would rather have consistency with our guns than to make special exceptions - if it's a pistol, SMG, or a revolver, it can fit in a bag. If it's not, it can't. I'll wait for a maintainer to give their piece on it.

Reply:
Portal gun can be an exception